cura/CuraApplication.py

```diff
@@ -1442,6 +1442,13 @@ def resetAll(self):
     # Single build plate
     @pyqtSlot()
     def arrangeAll(self) -> None:
+        self._arrangeAll(False)
+
+    @pyqtSlot()
+    def arrangeAllInGrid(self) -> None:
+        self._arrangeAll(True)
+
+    def _arrangeAll(self, grid_arrangement: bool) -> None:
```

Comment:
Another booltrap. You can already see why this can cause confusion a few lines up `self._arrangeAll(False)`. Without knowing what the fucntion is exactly, there is no way to figure out what the False means.

Reply:
fixed in https://github.com/Ultimaker/Cura/pull/15496/commits/118f49a0520bdd1a3983bbf33b49c3ed8a645fb1